src/main/java/com/openlattice/linking/RealtimeLinkingApi.java

```diff
@@ -49,4 +53,17 @@
 
     @GET( BASE + MATCHED + LINKING_ID_PATH )
     Set<MatchedEntityPair> getMatchedEntitiesForLinkingId( @Path( LINKING_ID ) UUID linkingId );
+
+    /**
+     * Get counts of entities that haven't been linked yet, grouped by entity set
+     *
+     * @param entitySetIds A set of normal entity set ids
+     * @return A map with entitySetIds as keys, and as values a count of the entityKeyIds that haven't been linked yet.
```

Comment:
This is a liiiiittle bit misleading, since it technically returns the number of entityKeyIds that should be queued for linking, even if they have already been linked (which will likely be the case in many/most cases)

Reply:
fixed !